From 6ccfaee225f119f3511b431f80ce1d7d66b15551 Mon Sep 17 00:00:00 2001
From: Xiangyou Xie <xiexiangyou@huawei.com>
Date: Fri, 19 Feb 2021 15:14:35 +0800
Subject: [PATCH] config: enable CONFIG_CPU_IDLE_GOV_HALTPOLL and
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 6ccfaee225f119f3511b431f80ce1d7d66b15551

 CONFIG_HALTPOLL_CPUIDLE for arm

hulk inclusion
category: config
bugzilla: 47727
CVE: NA

------------------------------

We enable haltpoll by default for the improvement of performance.
X86 has been supported. Now, we will provide it on ARM.

Signed-off-by: Xiangyou Xie <xiexiangyou@huawei.com>
Signed-off-by: Peng Liang <liangpeng10@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 02e5e8503e8a..72cb13ead2e8 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -526,12 +526,14 @@ CONFIG_CPU_IDLE_MULTIPLE_DRIVERS=y
 # CONFIG_CPU_IDLE_GOV_LADDER is not set
 CONFIG_CPU_IDLE_GOV_MENU=y
 CONFIG_CPU_IDLE_GOV_TEO=y
+CONFIG_CPU_IDLE_GOV_HALTPOLL=y
 CONFIG_DT_IDLE_STATES=y
 
 #
 # ARM CPU Idle Drivers
 #
 CONFIG_ARM_CPUIDLE=y
+CONFIG_HALTPOLL_CPUIDLE=y
 # CONFIG_ARM_PSCI_CPUIDLE is not set
 # end of ARM CPU Idle Drivers
 # end of CPU Idle
-- 
2.26.2

